Modules/Loadable/Markups/Widgets/Resources/UI/qMRMLMarkupsInteractionHandleWidget.ui

```diff
@@ -32,71 +32,303 @@
+      <set>Qt::AlignCenter</set>
+     </property>
+    </widget>
+   </item>
+   <item row="3" column="7">
+    <widget class="QCheckBox" name="rotateYCheckBox">
+     <property name="text">
+      <string/>
+     </property>
+    </widget>
+   </item>
+   <item row="5" column="3">
+    <widget class="QCheckBox" name="scaleVisibilityCheckBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
-      <string>Visibility:</string>
+      <string/>
      </property>
     </widget>
    </item>
-   <item row="3" column="1" colspan="2">
-    <widget class="ctkSliderWidget" name="interactionHandleScaleSlider">
-     <property name="singleStep">
-      <double>0.100000000000000</double>
+   <item row="1" column="6">
+    <widget class="QCheckBox" name="translateXCheckBox">
+     <property name="text">
+      <string/>
      </property>
-     <property name="maximum">
-      <double>20.000000000000000</double>
+    </widget>
+   </item>
+   <item row="5" column="6">
+    <widget class="QCheckBox" name="scaleXCheckBox">
+     <property name="text">
+      <string/>
      </property>
-     <property name="suffix">
-      <string> %</string>
+    </widget>
+   </item>
+   <item row="5" column="7">
+    <widget class="QCheckBox" name="scaleYCheckBox">
+     <property name="text">
+      <string/>
      </property>
     </widget>
    </item>
-   <item row="2" column="1" colspan="2">
+   <item row="13" column="0">
+    <widget class="QLabel" name="label_2">
+     <property name="text">
+      <string>Size:</string>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
-      <widget class="QCheckBox" name="translateVisibilityCheckBox">
-       <property name="text">
-        <string>Translate</string>
+      <widget class="QCheckBox" name="overallVisibilityCheckBox">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
        </property>
-      </widget>
-     </item>
-     <item>
-      <widget class="QCheckBox" name="rotateVisibilityCheckBox">
        <property name="text">
-        <string>Rotate</string>
+        <string/>
        </property>
       </widget>
      </item>
      <item>
-      <widget class="QCheckBox" name="scaleVisibilityCheckBox">
+      <widget class="QPushButton" name="moreOptionsCheckBox">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
        <property name="text">
-        <string>Scale</string>
+        <string>More options...</string>
```

Comment:
Nitpick: The widget is very crowded on the left side and there is a lot of space on the right. Could you try moving the "More options" button to the top-right corner? If it does not look good or not feasible for some reason then you could also try a tall double-arrow button (similarly to "Module paths" in application settings / modules) to show/hide that section.